api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java

```diff
@@ -56,7 +75,7 @@ public CohortDefinition getClientsWhoNewlyInitiatedPrep() {
             genericCohortQueries.getPatientAgeBasedOnPrepStartDate(15, 200),
             "startDate=${startDate},endDate=${endDate},location=${location}"));
```

Comment:
@TinoMuianga This fix is not correct! The Prep New Requirements, `PREP_NEW_FR2`  talks about **"clients included are the adults (>=15 years old) who initiated the treatment"** `PREP_NEW_FR4`, but RM PREP does not use the age verification. Please refer to the Prep New Requirements and apply a specific correction for this report